### PR TITLE
Uses rubocop as Ruby styleguide

### PR DESCRIPTION
**Pull request by FrankKair**
**Fixes issue:** 
No particular issue.

**Changes:**
Uses Rubocop to lint all Ruby files.

### CODE REVIEW

File: code/computational_geometry/2d_line_intersection/line.rb [hunk 1]
```diff
@@ -1,6 +1,5 @@
 class Line_interserction
```
Comment (Vitorvgc) [inline code/computational_geometry/2d_line_intersection/line.rb]: Classes should be in `CamelCase`.

File: code/search/binary_search/binary_search.rb [hunk 2]
```diff
@@ -1,27 +1,22 @@
-=begin
-	Part of Cosmos by OpenGenus Foundation
-=end
+#   Part of Cosmos by OpenGenus Foundation
 
 def BinarySearch(arr, l, r, x)
```
Comment (Vitorvgc) [inline code/search/binary_search/binary_search.rb]: Methods should be in `snake_case`.

File: code/sorting/insertion_sort/insertion_sort.rb [hunk 3]
```diff
@@ -1,21 +1,19 @@
-=begin
-    Part of Cosmos by OpenGenus Foundation
-=end
+#     Part of Cosmos by OpenGenus Foundation
 
 def InsertionSort(arr)
```
Comment (Vitorvgc) [inline code/sorting/insertion_sort/insertion_sort.rb]: Methods should be in `snake_case`.

File: code/search/binary_search/binary_search.rb (hunk 2 above)
Comment (FrankKair) [reply code/search/binary_search/binary_search.rb]: Done

File: code/computational_geometry/2d_line_intersection/line.rb (hunk 1 above)
Comment (FrankKair) [reply code/computational_geometry/2d_line_intersection/line.rb]: Done

File: code/sorting/insertion_sort/insertion_sort.rb (hunk 3 above)
Comment (FrankKair) [reply code/sorting/insertion_sort/insertion_sort.rb]: Done

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**